Sources/Testing/Running/Runner.swift

```diff
@@ -314,8 +318,13 @@ extension Runner {
       }
     }
 
+    // Figure out how to name child tasks.
+    func taskNamer(_ childGraph: Graph<String, Plan.Step?>) -> String? {
+      childGraph.value.map { $0.test.humanReadableName() }
```

Comment:
With the new prefix, this will end up being something like this, right?

> [Swift Testing] "Foo bar"

if the test has a display name `"Foo bar"`. Display names in particular might be somewhat hard to parse out of context and understand that it represents an individual test. It's more clear when it's a function name, but I worry the display name case will be unclear. Maybe refine the prefix further and have

> [Swift Testing] Test: "Foo bar"
…or…
> [Swift Testing] Test: anotherExample()

Reply:
Changed.